scripts/compare-fscontent/compare-fscontent.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/invoker"
+	"github.com/nspcc-dev/neofs-contract/rpc/container"
+	"github.com/nspcc-dev/neofs-contract/rpc/netmap"
+	"github.com/nspcc-dev/neofs-contract/rpc/nns"
+	"github.com/pmezard/go-difflib/difflib"
+	"github.com/urfave/cli"
+)
+
+func initClient(addr string, name string) (*rpcclient.Client, uint32, error) {
+	c, err := rpcclient.New(context.Background(), addr, rpcclient.Options{})
```

Comment:
does this cli lib has its own `Context` to pass it here?

Reply:
No.